src/core/scene/a-scene.js

```diff
@@ -453,18 +433,19 @@ module.exports.AScene = registerElement('a-scene', {
     },
 
     resize: {
-      value: function () {
+      value: function (ll) {
         var camera = this.camera;
         var canvas = this.canvas;
         var embedded = this.getAttribute('embedded') && !this.is('vr-mode');
         var size;
-        var isEffectPresenting = this.effect && this.effect.isPresenting;
+        var vrDevice = this.renderer.vr.getDevice();
```

Comment:
we can initialize these two variables out of the declaration block since they have some logic